### PR TITLE
12.0 mig account payment sale

### PR DESCRIPTION
**Pull request by mourad-ehm**

### CODE REVIEW

File: account_payment_sale/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    'website': 'https://github.com/OCA/bank-payment',
+    'depends': [
+        'sale',
+        'account_payment_partner',
+    ],
+    'data': [
+        'views/sale_order_view.xml',
+        'views/sale_report_templates.xml',
+    ],
+    'auto_install': True,
```
Comment (ValentinVinagre) [inline account_payment_sale/__manifest__.py:23]: It's correct?

File: account_payment_sale/views/sale_order_view.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+<record id="view_order_form" model="ir.ui.view">
+    <field name="name">account_payment_sale.sale_order.form</field>
+    <field name="model">sale.order</field>
+    <field name="inherit_id" ref="sale.view_order_form"/>
+    <field name="arch" type="xml">
+        <field name="payment_term_id" position="after">
+            <field name="payment_mode_id"
+                domain="[('payment_type', '=', 'inbound')]"
+                widget="selection"/>
```
Comment (ValentinVinagre) [inline account_payment_sale/views/sale_order_view.xml]: Can change this widget by:
```suggestion
                options="{'no_open': True, 'no_create': True}"
```
I suggest this change why we have a lot of client what have a lot payment modes and the selection widget only show part of them.

**Review by ValentinVinagre — COMMENTED** [review]
Hello,
I have indicated some comments, if you think they are necessary you could incorporate them.
A greeting.

File: account_payment_sale/__manifest__.py (hunk 1 above)
Comment (pedrobaeza) [reply account_payment_sale/__manifest__.py:23]: Well, I would leave it for now.

**Comment by pedrobaeza** [comment]
Merging this one as well and continuing for bingo, hehe!

**Pull request merged**